### PR TITLE
Update the existing "About DuckDuckGo" screen to match the new design

### PR DESCRIPTION
**Pull request by anikiki**
Task/Issue URL: https://app.asana.com/0/0/1204670283311481/f

### Description
Updated the `About DuckDuckGo` screen.

### Steps to test this PR

- [ ] Install from this branch.
- [ ] Go to the `Settings` screen and tap on `About`.
- [ ] Notice the updated `About DuckDuckGo` screen. The last 3 items were moved from `Settings`. Check they behave the same.

### UI changes
| Before  | After |
| ------ | ----- |
|about_ddg_before|about_ddg_after|

### CODE REVIEW

**Comment by anikiki** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #3226** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3226" target="_blank">Graphite</a> 
    * **PR #3230** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3230" target="_blank">Graphite</a> 
      * **PR #3246** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3246" target="_blank">Graphite</a> 
        * **PR #3248** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3248" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/3248?utm_source=stack-comment).

File: app/src/main/java/com/duckduckgo/app/about/AboutDuckDuckGoActivity.kt [hunk 1]
```diff
@@ -19,31 +19,144 @@ package com.duckduckgo.app.about
...
 
 @InjectWith(ActivityScope::class)
 class AboutDuckDuckGoActivity : DuckDuckGoActivity() {
 
+    private val viewModel: AboutDuckDuckGoViewModel by bindViewModel()
     private val binding: ActivityAboutDuckDuckGoBinding by viewBinding()
 
+    private val feedbackFlow = registerForActivityResult(FeedbackContract()) { resultOk ->
+        if (resultOk) {
+            Toast.makeText(this, R.string.thanksForTheFeedback, Toast.LENGTH_LONG).show()
```
Comment (malmstein) [inline app/src/main/java/com/duckduckgo/app/about/AboutDuckDuckGoActivity.kt:51]: Can you make this a Snackbar instead?

File: app/src/main/java/com/duckduckgo/app/about/AboutDuckDuckGoViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+    fun viewState(): StateFlow<ViewState> {
+        return viewState
+    }
+
+    fun commands(): Flow<Command> {
+        return command.receiveAsFlow()
+    }
+
+    fun start() {
```
Comment (malmstein) [inline app/src/main/java/com/duckduckgo/app/about/AboutDuckDuckGoViewModel.kt:70]: Can you use a more descriptive name here? Start doesn't really explain what happens here.

**Review by malmstein — APPROVED** [review]
Just a couple nits about naming. Rest looks good!

File: app/src/main/java/com/duckduckgo/app/about/AboutDuckDuckGoActivity.kt (hunk 1 above)
Comment (anikiki) [reply app/src/main/java/com/duckduckgo/app/about/AboutDuckDuckGoActivity.kt]: Replaced the Toast with a Snackbar 👍

File: app/src/main/java/com/duckduckgo/app/about/AboutDuckDuckGoViewModel.kt (hunk 2 above)
Comment (anikiki) [reply app/src/main/java/com/duckduckgo/app/about/AboutDuckDuckGoViewModel.kt]: Updated the name 👍

**Pull request merged**